### PR TITLE
new(NRQL comments): new docs, & link

### PR DESCRIPTION
**Pull request by zuluecho9**
NR-98268

### CODE REVIEW

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx [hunk 1]
```diff
@@ -208,6 +208,35 @@ Another great way to explore your data is to go into any existing dashboards you
   Charts built with NRQL will have **View query** as an option. You can then edit and customize that query to see how your changes affect the resulting visualization.
 </figcaption>
 
+## Comments [#comments]
+
+When writing a NRQL query, you can add comments to a line or section of text. Comments can help your team members better understand and use the queries you make.
```
Comment (jeff-colucci) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx:213]: What do you think about merging the two sentences here? "section of text that help your team members" Just a thought.

**Review by jeff-colucci — DISMISSED** [review]
Looks good to me!

File: src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx [hunk 2]
```diff
@@ -154,6 +154,15 @@ NRQL's basic rules are as follows:
         NRQL does not have the equivalent of the SQL `JOIN` clause, but you can [simulate a `JOIN` with custom attributes](/docs/insights/new-relic-insights/using-new-relic-query-language/simulate-sql-join-functions-insights).
       </td>
     </tr>
+    <tr>
+      <td>
+        Comments
+      </td>
+
+      <td>
+        You can add non-working comments to your NRQL queries. [Learn more about comments](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-syntax-clauses-functions#comments).
```
Comment (mlobiondo-newrelic) [inline src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works.mdx:163]: I think "non-working" sounds strange here, I would omit.  Other than that, this looks good!  Thank you!

**Review by mlobiondo-newrelic — APPROVED** [review]

**Review by mlobiondo-newrelic — APPROVED** [review]

**Comment by mlobiondo-newrelic** [comment]
Looks good.  QLang is ready for this to be merged whenever you're ready, @zuluecho9.

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**